### ISSUE TITLE
Zen Pinball 1 & 2: Audio cuts out during gameplay

### ISSUE THREAD

**Issue by Shoegzer** [RPCS3/rpcs3#7724]
Zen Pinball 1 & 2 go ingame with full audio, but eventually (usually around 1-5 minutes of gameplay) all sounds stop completely.  This issue has existed for years and may have always been, but continues as of commit deb6bd3e25f94311d1177df578e388111aa0f5e2.

Update: I've noticed that setting the PPU Decoder to "Interpreter (fast)" fixes this problem (although then the game naturally plays too slow).  Setting Preferred SPU Threads to "1" also fixes the problem without the speed hit.  I'm guessing this may be related either to timing between the PPU and SPU, or to thread synchronization. In any event I'm willing to close the issue if others agree - I just want to be sure this at least gets seen before I do in the event that it provides a clue into a wider issue.  @Nekotekina how do you feel about this?
 
Latest test was performed on system with linux mint 19.3 with kernel 5.3 compiled with gcc 9.2.1.  Tested primarily with OpenAL but it occurs with other audio backends too.

Log, stopped once issue occurred: [RPCS3.log](https://github.com/RPCS3/rpcs3/files/4302532/RPCS3.log)

**Comment by Asinin3** [comment]
Still an issue?

**Comment by Shoegzer** [comment]
Yes, as confirmed with commit 42e3add6a1a5e2a28b5e83389ebea7540606cb0d.  I was able to reproduce the issue almost immediately with ZP1, and while I haven't after testing for 5-10 minutes with ZP2 I assume it would eventually as well.

Again I'm not opposed to closing this since limiting SPU threads fixes the problem, though there may be value in leaving it open in the event it might help expose a greater issue.

**Comment by legluondunet** [comment]
Hello,
this bug is still present.
After some  minutes playing Starwars Pinball, no more sound.
I can reproduce this bug all the time.

log joined.
[rpcs3.log](https://github.com/RPCS3/rpcs3/files/10328890/rpcs3.log)

**Comment by digant73** [comment]
@Shoegzer is that still an issue?

**Comment by Shoegzer** [comment]
It doesn't appear to be.  I tested both ZP1 and ZP2 for quite a while on default settings with no sound dropouts, so I'll close this.  If anyone encounters them again I can reopen.

**Comment by legluondunet** [comment]
I can no more reproduce this issue, thanks.
